src/js/components/VoterGuide/VoterGuideBallotItemCompressed.jsx

```diff
@@ -25,8 +25,13 @@ export default class VoterGuideBallotItemCompressed extends Component {
           />
         ) : (
           <VoterGuideOfficeItemCompressed
-            {...this.props}
+            ballotItemDisplayName={this.props.ballot_item_display_name}
```

Comment:
Please see my comments here: https://github.com/wevote/WebApp/pull/3479/files

Reply:
Hello @MGarc758 the changes you made in your other approved pull request need to be included in this pull request too.